### PR TITLE
[SPARK-26207][doc]add PowerIterationClustering (PIC) doc in 2.4 branch

### PR DESCRIPTION
**Pull request by huaxingao**
## What changes were proposed in this pull request?

Add PIC doc in 2.4

## How was this patch tested?

Manually tested

### CODE REVIEW

**Comment by huaxingao** [comment]
@felixcheung Could you please review?

**Comment by gatorsmile** [comment]
cc @srowen

**Comment by huaxingao** [comment]
@srowen It's not in master yet. The PR is here
https://github.com/apache/spark/pull/23072

**Review by felixcheung — COMMENTED** [review]
this too 
https://github.com/apache/spark/pull/23072/files#diff-b0703c5563039bf2a97758ea0c941840L67

how about this (I know this isn't doc/example change)
https://github.com/apache/spark/pull/23072/files#diff-001c02af100eec4d8ba87335c1591c08R100

File: docs/ml-clustering.md [hunk 1]
```diff
@@ -265,3 +265,38 @@ Refer to the [R API docs](api/R/spark.gaussianMixture.html) for more details.
 </div>
 
 </div>
+
+## Power Iteration Clustering (PIC)
+
+Power Iteration Clustering (PIC) is  a scalable graph clustering algorithm
+developed by <a href=http://www.icml2010.org/papers/387.pdf>Lin and Cohen</a>.
```
Comment (shahidki31) [inline docs/ml-clustering.md]: Hi @huaxingao ,  It seems  the link is not accessible now. This link (http://www.cs.cmu.edu/~frank/papers/icml2010-pic-final.pdf)  seems working.

Comment (huaxingao) [reply docs/ml-clustering.md]: @shahidki31 Changed. Thanks!

**Comment by srowen** [comment]
Merged to 2.4

**Pull request closed without merging**